views/results.ejs

```diff
@@ -2,4 +2,14 @@
 
 <h1>These people do:</h1>
 
+<form>
+  <label class="Display1 ml3">
```

Comment:
No need to follow a form convention using <label>, just use an `h` element and class appropriately. Probably an h2 or h3